newIDE/app/src/MainFrame/UseMultiplayerLobbyConfigurator.js

```diff
@@ -46,7 +46,13 @@ export const useMultiplayerLobbyConfigurator = (): UseMultiplayerLobbyConfigurat
           await registerGame(
             getAuthorizationHeader,
             profile.id,
-            getDefaultRegisterGamePropertiesFromProject({ project })
+            getDefaultRegisterGameProperties({
+              projectId: project.getProjectUuid(),
+              projectName: project.getName(),
+              projectAuthor: project.getAuthor(),
+              // Assume the project is not saved at this stage.
+              isProjectSaved: false,
```

Comment:
I'm a bit uneasy that we went for "isProjectSaved" and "markGameAsSavedIfRelevant" instead of the notion of "draft", which is clearer for the reader. ("why do you need to know if the project is saved??? Let me open this function... ah ok it's actually to check if it's a draft or not" vs "ok you're tracking the draft status or not, make sense")